db/view/view.cc

```diff
@@ -1625,25 +1625,26 @@ get_view_natural_endpoint(
         }
     }
```

Comment:
> ERMs 

That's one of our worst acronyms

Reply:
We should pay Scylla employees for fast-typing lessons, so they won't have to use acronyms in their code :-)

Scylla could also sponsor the AAAAAAA - the American Association Against Abuse of Abbreviations And Acronyms.

Reply:
> We should pay Scylla employees for fast-typing lessons, so they won't have to use acronyms in their code :-)
> 

And for wider monitors!